oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -533,64 +535,12 @@ protected static String getReleaseFilename() {
      * @return Mixed case family
      */
     private static String filenameToFamily(String name) {
-        switch (name.toLowerCase()) {
-        // Handle known special cases
-        case "":
+        Properties filenameProps = FileUtil.readPropertiesFromFilename(FILENAME_PROPERTIES);
```

Comment:
If the name is empty we can skip reading the properties, so this can be moved down into the "else" block.